### PR TITLE
Add docs for Sandbox

### PR DESCRIPTION
**Pull request by xiaoyongzhu**
Adding docs for Feathr sandbox

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -74,6 +74,26 @@ If you want to set up everything manually, you can checkout the [Feathr CLI depl
 | [Fraud Detection Demo](./samples/fraud_detection_demo.ipynb)                        | An example to demonstrate Feature Store using multiple data sources such as user account and transaction data.                        | Azure Synapse, Databricks, Local Spark |
 | [Product Recommendation Demo](./samples/product_recommendation_demo_advanced.ipynb) | Feathr Feature Store example notebook with a product recommendation scenario                                                          | Azure Synapse, Databricks, Local Spark |
 
+## 🏃 Getting Started with Feathr
+
+The best way to try out Feathr is to use the [Feathr Sandbox](https://feathr-ai.github.io/feathr/quickstart_local_sandbox.html) which is a self-contained container with most of Feathr's capabilities. To use it, simply run this command:
```
Comment (Yuqing-cat) [inline docs/README.md:79]: `The best way` seems to be a risky statement. Shall we confirm it with other team members?

Comment (xiaoyongzhu) [reply docs/README.md]: changed to `easiest`

File: docs/quickstart_local_sandbox.md [hunk 2]
```diff
@@ -21,15 +21,25 @@ The Sandbox is ideal for:
...
 
 ```bash
 # 80: Feathr UI 8000: Feathr API 8888: Jupyter 8080: VsCode 7080: Interpret
-docker run -it --rm -p 8888:8888  -p 8000:8000 -p 80:80 -p 8080:8080 -p 7080:7080 --env CONNECTION_STR="Server=" --env API_BASE="api/v1" --env FEATHR_SANDBOX=True -e GRANT_SUDO=yes feathrfeaturestore/feathr-sandbox
+docker run -it --rm -p 8888:8888  -p 8000:8000 -p 80:80 -p 8080:8080 -p 7080:7080 --env API_BASE="api/v1" --env FEATHR_SANDBOX=True -e GRANT_SUDO=yes feathrfeaturestore/feathr-sandbox
 ```
 
 It should pop up a Jupyter link in `http://127.0.0.1:8888/`. Double click on the notebook file to start the Jupyter Notebook, and you should be able to see the Feathr sample notebook. Click the triangle button on the Jupyter notebook and the whole notebook will run locally.
 
+If you see some errors like below, simply change the `-p 80:80` part to `-p <new port>:80` so the Feathr UI will be redirected to the new port.
```
Comment (blrchen) [inline docs/quickstart_local_sandbox.md:39]: Suggest use a port great than 1024 for registry ui, as port les than 1024 is likely failed to use in most system.

**Review by blrchen — APPROVED** [review]
:shipit:

**Pull request merged**